src/lndmobile/index.ts

```diff
@@ -196,13 +196,13 @@ export const sendPaymentSync = async (paymentRequest: string, amount?: Long, tlv
 };
 
 
-export const sendPaymentV2Sync = (paymentRequest: string, amount?: Long, tlvRecordName?: string | null, multiPath?: boolean): Promise<lnrpc.Payment> => {
+export const sendPaymentV2Sync = (paymentRequest: string, amount?: Long, pay_amount: Long, tlvRecordName?: string | null, multiPath?: boolean): Promise<lnrpc.Payment> => {
```

Comment:
Would you mind changing the variable name to camelCase to match the code styling?

```suggestion
export const sendPaymentV2Sync = (paymentRequest: string, amount?: Long, payAmount: Long, tlvRecordName?: string | null, multiPath?: boolean): Promise<lnrpc.Payment> => {